test/WebJobs.Extensions.DurableTask.Analyzers.Test/Orchestrator/DateTimeAnalyzerTests.cs

```diff
@@ -31,57 +31,60 @@ public void dateTimeNow()
 
         private const string ExpectedFix = @"
     using System;
-    using System.Collections.Generic;
     using System.Threading.Tasks;
     using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
 
     namespace VSSample
     {
         public static class HelloSequence
         {
-            [FunctionName(""E1_HelloSequence"")]
-            public static async Task<List<string>> Run(
+            [FunctionName(""DateTimeAnalyzerTestCases"")]
+            public static async Task Run(
             [OrchestrationTrigger] IDurableOrchestrationContext context)
             {
-                context.CurrentUtcDateTime;
+                var dateTime = context.CurrentUtcDateTime;
             }
         }
     }";
 
         [TestMethod]
-        public void DateTime_InMethod_NonIssueCalls()
+        public void DateTime_NotOrchestrator_NoDiagnosticTestCases()
         {
             var test = @"
     using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
 
     namespace VSSample
     {
-        public static class DateTimeNowExample
+        public static class HelloSequence
         {
-            [FunctionName(""E1_HelloSequence"")]
-            " + allTests;
+            [FunctionName(""DateTimeAnalyzerTestCases"")]
+            " + allNonDiagnosticTestCases;
 
             VerifyCSharpDiagnostic(test);
         }
         
         [TestMethod]
-        public void DateTimeInOrchestrator_Now_Namespace()
+        public void DateTime_Now_WithNamespace()
```

Comment:
Since all of the orchestration pieces of code are almost identical except for which value goes on the other side of `var dateTime=`, it may be possible to parameterize these tests.

Reply:
That makes sense, considering these are all just strings.

Reply:
I'm going to save this suggestion for a future test refactor and not update the tests again in this PR.